pkg/skaffold/deploy/cloudrun/status.go

```diff
@@ -344,9 +344,13 @@ func (r *runServiceResource) getTerminalStatus(crClient *run.APIService) (*run.G
 	return ready, nil
 }
 func (r *runServiceResource) reportSuccess() {
-	if r.url != "" {
-		eventV2.CloudRunServiceReady(r.path, r.url, r.latestRevision)
+	url := r.url
+	if url == "" {
```

Comment:
I think it'll be better to check if the service definition contains the feature flags:  
```
run.googleapis.com/launch-stage: 'BETA'
run.googleapis.com/default-url-disabled: 'true'
```

Reply:
I don't want to hard-code this to that specific feature (I also definitely don't want to base it on the launch stage because that'll be removed when the feature goes GA). I think it makes more sense to trigger the event any time the service goes ready regardless of whether there's a URL or not, and as this feature shows a URL being present is not a requirement for a service going ready.